src/vaadin-message-input.js

```diff
@@ -41,8 +55,8 @@ class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
           margin: 0;
         }
       </style>
-      <vaadin-text-area placeholder="Message"></vaadin-text-area>
-      <vaadin-button theme="primary contained">Send</vaadin-button>
+      <vaadin-text-area value="{{text}}" placeholder="Message"></vaadin-text-area>
+      <vaadin-button theme="primary contained" on-click="submitMessage">Send</vaadin-button>
```

Comment:
Please create `vaadin-message-text-area` and `vaadin-message-button` - see https://github.com/vaadin/vaadin-messages/issues/28#issuecomment-779793218.

Reply:
I don't think it's meaningful to do in the scope of this PR.

I'll update #28 to have the responsibility for all these three cases.